content/08-mongodb/11-mongodb-atlas-setup.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: 'Introduction to provisioning MongoDB Atlas'
+metaTitle: "MongoDB Atlas Setup | Prisma's Data Guide"
+metaDescription: "In this guide, we'll cover provisioning MongoDB Atlas."
+metaImage: '/content/mongodb/header.png'
+authors: ['alexemerich']
+---
+## Introduction
+
+MongoDB Atlas is MongoDB's cloud database service. It offers data distribution and mobility across major cloud providers AWS, Azure, and Google Gloud. It is popular for its built-in automation for resource and workload optimization. 
```

Comment:
```suggestion
MongoDB Atlas is MongoDB's cloud database service. It offers data distribution and mobility across major cloud providers like AWS, Azure, and Google Cloud. It is popular for its built-in automation for resource and workload optimization. 
```

The "like" is just an option.  I think you're missing a word there but something else might be better.